bonfire/namespaces.py

```diff
@@ -83,17 +83,6 @@ def status(self):
     def reserved(self):
         return self.annotations.get("reserved", "false") == "true"
 
-    @property
```

Comment:
Could we:
1. Edit the 'pool_type' function in this class and have it return `self.labels.get("pool")`
2. Move the `def is_reservable` function to be defined below the `def pool_type` function (I just like to have functions that are used in other functions defined "higher up" in the file when possible for readability)
2. Keep the `is_reservable` property and have it return `self.pool_type is not None`